src/System.Private.CoreLib/shared/System/Threading/WaitHandle.cs

```diff
@@ -2,30 +2,24 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-/*=============================================================================
-**
-**
-**
-** Purpose: Class to represent all synchronization objects in the runtime (that allow multiple wait)
-**
-**
-=============================================================================*/
-
 using System.Diagnostics;
-using System.Diagnostics.CodeAnalysis;
-using System.Runtime.InteropServices;
-using Internal.Runtime.Augments;
 using Microsoft.Win32.SafeHandles;
 
 namespace System.Threading
 {
     public abstract partial class WaitHandle : MarshalByRefObject, IDisposable
     {
-        public const int WaitTimeout = 0x102;
-        protected static readonly IntPtr InvalidHandle = new IntPtr(-1);
+        internal const int WaitAbandoned = 0x80;
+        internal const int WaitTimeout = 0x102;
```

Comment:
I'm thinking about wrapping these values as an `enum` for use from `WaitSubsystem`.